### ISSUE TITLE
Error using cnvgrib on WCOSS2

### ISSUE THREAD

**Issue by kgerheiser** [NOAA-EMC/NCEPLIBS-grib_util#135]
From an issue reported by @JunDu-NOAA 

The current version (1.2.2) of the grib conversion cnvgrib in the wcoss2 doesn't work for the sref system. I got the following error: 

```
0.655 + /apps/ops/prod/libs/intel/19.1.3.304/grib_util/1.2.2/bin/cnvgrib -g12 -p40 -nv sref_arw.t09z.pgrb212.ctl sref_arw.t09z.pgrb212.ctl.grib2:
  Unrecognized Ensemble type =            0
 pds2pdtens: Couldn:t construct  PDS Template
  Unrecognized Ensemble type =            0
 pds2pdtens: Couldn:t construct  PDS Template
  Unrecognized Ensemble type =            0
``` 

I did a test: 

After I copied the wcoss grib_util/1.1.1 over to wcoss2, it works fine.

**Comment by edwardhartnett** [comment]
There are many versions between 1.1.1 and 1.2.2. Do you have any idea which version introduced the bug?
How big are these test files?

**Comment by JunDu-NOAA** [comment]
I don't know exactly. Maybe can check which version is currently used by
the operational SREF in WCOSS.
Or you point me version, I can test it.

Jun

**Comment by kgerheiser** [comment]
This was fixed with #139
